wiki/Tournaments/CFC/3/en.md

```diff
@@ -0,0 +1,201 @@
+---
+tags:
+- Catch French Cup 3
+- CFC3
+---
+# Catch French Cup 3
+
+![CFC 3 Logo](img/logo.png)
+
+The **Catch French Cup 3 (***CFC 3***)** is a one-on-one French osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](/users/6177263). It is the 3rd installment of the Catch French Cup.
+
+## Tournament Schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2018-08-15/2018-09-02 |
+| Live Drawings | 2018-09-01 (19:00 UTC) |
```

Comment:
`Live Drawings` happen before `Registration Phase` end?

Reply:
It happened before Registration Phase indeed according to the Organiser